descqa/DensityVersusExtinction.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import unicode_literals, absolute_import, division
+import os
+import numpy as np
+from scipy.stats import binned_statistic
+from .base import BaseValidationTest, TestResult
+from .plotting import plt
+import healpy as hp
+
+__all__ = ['DensityVersusExtinction']
+
+def create_hp_map(ra, dec, nside):
+    """
+    Auxiliary function to generate HEALPix maps from catalogs.
+    It reads the ra and dec in degrees and returns a HEALPix map
+    """
+    pixnums = hp.ang2pix(nside, np.pi/2-np.radians(dec), np.radians(ra))
+    return np.bincount(pixnums, minlength=hp.nside2npix(nside))
+
+class DensityVersusExtinction(BaseValidationTest):
```

Comment:
Will this class better be named as `DensityVersusSkyPosition` or something along the line? In particular, since `validation_map_filename` and `xlabel` are both configurable options, one can supply another validation map that is not related to extinction, right? Then it seems to me that the class name is too restrictive?

Reply:
You are totally right. Thanks!